version/distro/distro.go

```diff
@@ -153,28 +153,6 @@ func DSMVersion() int {
 	})
 }
 
-// isUDMPro checks a couple of files known to exist on a UDM-Pro and returns
-// true if the expected content exists in the files.
-func isUDMPro() bool {
-	// This is a performance guardrail against trying to load both
-	// /etc/board.info and /sys/firmware/devicetree/base/soc/board-cfg/id when
-	// not running on Debian so we don't make unnecessary calls in situations
-	// where we definitely are NOT on a UDM Pro. In other words, the have() call
-	// is much cheaper than the two os.ReadFile() in fileContainsAnyString().
-	// That said, on Debian systems we will still be making the two
-	// os.ReadFile() in fileContainsAnyString().
-	if !have("/etc/debian_version") {
-		return false
-	}
-	if exists, err := fileContainsAnyString("/etc/board.info", "UDMPRO", "Dream Machine PRO"); err == nil && exists {
```

Comment:
All uses of `fileContainsAnyString` are gone, so remove that too.

Reply:
Removed.